index.html

```diff
@@ -1208,6 +1213,11 @@ <h2>Extract response headers</h2>
           </p>
         </li>
 
+        <li>
+          If <var>policy</var> is <a>stale</a>, then delete <var>policy</var>

```

Comment:
How does this enable stale-while-revalidate semantics?

Reply:
My intention here was to allow a stale policy to be used to send a single final report; by removing it from the cache here (but also returning it) the policy is used this time, but then needs to be reinstated by the origin server before another report will be sent.

This way, if the server stops sending NEL headers, then the last policy seen will be used to report errors up to 48h later (unless max_age comes first). Between 48h and max_age, just one last report will be sent. After max_age, nothing is sent at all.